motionplan/ptgGroupFrame.go

```diff
@@ -0,0 +1,193 @@
+package motionplan
```

Comment:
`ptgIK` is in motionplan because of the use of IK, and also needs to import motionplan/tpspace.

This file needs to be able to import the ptgIK constructor, and can't import motionplan without creating a loop. Thus it needs to also be in motionplan.

Teeeeechnically I don't think ptgIK uses anything private in motionplan, and may be able to move to tpspace

Reply:
If its possible then it might be a good idea to try to keep everything contained to the tpspace package

Reply:
This actually can't work; these need to be in `motionplan`.

`motionplan` imports `tpspace`, so the IK frame that uses things in `motionplan` can't be in `tpspace` since `tpspace` can't import `motionplan`.

Since this file in turn uses that IK frame, so it also needs to be somewhere that can import `motionplan`.